akka-docs/rst/scala/code/docs/persistence/PersistenceMultiDocSpec.scala

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
```

Comment:
Is it just for new files of for all of the files I touched? 
I would imagine it has to be global akka find/replace, not peace-meal per-project approach?